wiki/People/Tournament_Committee/en.md

```diff
@@ -6,35 +13,53 @@ Their responsibilities include:
 
 - Managing reports from the [tournament reports form](https://pif.ephemeral.ink/tournament-reports)
 - Observing the general tournament scene and advising the [account support team](/wiki/People/Account_support_team) on a case-by-case basis
-- Regularly reviewing the [Official tournament support](/wiki/Tournaments/Official_support) page for revisions where necessary
+- Regularly reviewing the [Official tournament support](/wiki/Tournaments/Official_support) page for revisions when necessary
 
 The panel was first announced to the public on July 22, 2021 through the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) in response to the rapidly growing community tournament scene in osu!.
 
+## Contest Committee
```

Comment:
Probably should be called `Mapping Contest Committee` as there are contests outside of mapping

Reply:
so when contests outside mapping come around, the "mapping contest committee" should *not* be consulted, but instead there should be a new subcommittee?

it sounds a bit silly

Reply:
> but instead there should be a new subcommittee

no, I'm not sure why this would be implied.

if these people only care about mapping contests then I think it'd be good to include that in the name

Reply:
I'm in favour of changing this to `Mapping Contest Committee` for clarity

Reply:
(it doesn't seem like committee people are in favour of this)